modules/custom/inope_policy_review/inope_policy_review.module

```diff
@@ -173,7 +165,7 @@ function inope_policy_review_page($node) {
     // Build a table of results from other users.
     $build['table'] = array(
       '#type' => 'markup',
-      '#markup' => inope_policy_review_table($node, $account),
+      '#markup' => views_embed_view('policies', 'block_1'),
```

Comment:
views_embed_view returns HTML to be displayed on the page the render array is used as we have multiple items to display (either the form+view or message+view).

Do you have another method in mind you would like me to try?